src/server/pachyderm_test.go

```diff
@@ -5332,39 +5332,48 @@ func TestMaxQueueSize(t *testing.T) {
 			Transform: &pps.Transform{
 				Cmd: []string{"bash"},
 				Stdin: []string{
-					"sleep 10",
+					"sleep 5",
 				},
 			},
 			Input: client.NewAtomInput(dataRepo, "/*"),
 			ParallelismSpec: &pps.ParallelismSpec{
 				Constant: 2,
 			},
 			MaxQueueSize: 1,
+			ChunkSpec: &pps.ChunkSpec{
+				Number: 10,
+			},
```

Comment:
So with 10 chunks it seems like you should expect chunks of size 2, which is big enough to trip the bug, but just barely. Wouldn't this be a little more likely to fail if we set 2 chunks and thus had chunks of size 10?

Reply:
That number refers to the number of datums per chunk, so this spec would be 10 datums per chunk.

Reply:
A good point, I retract my previous question.